assets/css/lsx-wetu-importer.css

```
@@ -1,199 +1,258 @@
+  display: none;
+}
+body .to-wrapper #posts-filter .check-column input, body .to-wrapper #import-list .check-column input {
+  margin: 0 0 0 8px;
+}
+body .to-wrapper #posts-filter thead .sorting_asc::after, body .to-wrapper #posts-filter thead .sorting_desc::after, body .to-wrapper #posts-filter tfoot .sorting_asc::after, body .to-wrapper #posts-filter tfoot .sorting_desc::after, body .to-wrapper #import-list thead .sorting_asc::after, body .to-wrapper #import-list thead .sorting_desc::after, body .to-wrapper #import-list tfoot .sorting_asc::after, body .to-wrapper #import-list tfoot .sorting_desc::after {
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  font-size: 20px;
+  line-height: 1;
+  font-family: dashicons;
```

Biome (1.9.4)

[error] Generic font family missing.

Consider adding a generic font family as a fallback.
For examples and more information, see the MDN Web Docs


(lint/a11y/useGenericFontNames)